data/schema/units/specials.cfg

```diff
@@ -93,6 +93,13 @@
 	super="units/unit_type/attack/specials/~generic~"
 	{SIMPLE_KEY type string}
 [/tag]
+[tag]
+	name="damage_type"
+	max=infinite
+	super="units/unit_type/attack/specials/~generic~"
+	{SIMPLE_KEY type string}
+	{SIMPLE_KEY type_priority f_int}
```

Comment:
I do want to request that this be changed to `f_real`.

Reply:
this is done

Reply:
Actually, I just realized that `f_real` is also wrong. Your code doesn't use `typed_formula` to process the value in here, so it should probably be just `real`.

Reply:
> Actually, I just realized that `f_real` is also wrong. Your code doesn't use `typed_formula` to process the value in here, so it should probably be just `real`. (Maybe `s_real`, if other abilities have a habit of using `s_` prefixed types.)

`s_real`, or `real` then?

Reply:
I don't see any other `s_` prefixed types in the abilities. so let's go with `real`.

Reply:
done